### ISSUE TITLE
No such file or directory: 'requirements.txt'

### ISSUE THREAD

**Issue by emmercm** [Flexget/Flexget#910]
On Cygwin with Python 2.7.10 and pip 8.0.3:
```
$ pip install --no-cache-dir flexget
Collecting flexget
  Downloading FlexGet-1.2.467.tar.gz (2.0MB)
    100% |████████████████████████████████| 2.0MB 6.8MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-yf1tnj/flexget/setup.py", line 39, in <module>
        install_requires=load_requirements('requirements.txt'),
      File "/tmp/pip-build-yf1tnj/flexget/setup.py", line 20, in load_requirements
        with io.open(filename, encoding='utf-8') as reqfile:
    IOError: [Errno 2] No such file or directory: 'requirements.txt'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-yf1tnj/flexget
```

**Comment by stevezau** [comment]
dupe of #909

**Comment by emmercm** [comment]
Confirmed fixed in 1.2.468, thank you.
